src/NTLDataFormatter.py

```diff
@@ -6,7 +6,7 @@
 
 class NTLDataFormatter(FormatterInterface):
     def __init__(self):
-        pass
+        pass    #this is empty explain why
```

Comment:
Can change the comment to "intentionally empty constructor". Same for the comment in `src/SocrataDataFormatter.py`.

@Rodriguez-Carlos-BAH let us know if you prefer another explanation here.